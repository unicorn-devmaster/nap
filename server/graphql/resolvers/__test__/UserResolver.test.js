/* eslint-env jest */
describe('willReadUser', () => {
  it('should return error if session not exist', async () => {
    const { user : willReadUser } = require('../UserResolver')
    const context = { nap : {errors: []} }
    await willReadUser({ context }).catch(err => {
      expect(err).toMatchSnapshot()
    })
  })
})