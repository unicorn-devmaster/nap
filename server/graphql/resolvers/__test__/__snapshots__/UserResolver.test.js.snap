// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`willReadUser should return error if session not exist 1`] = `[Error: No session found]`;
